### ISSUE TITLE
Pressing 'Reset All' produces sound of last sim element with Studio 

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/ohms-law#165]
**Test device**
MacBook Air (m1 chip)

**Operating System**
12.2.1

**Browser**
Chrome

**Problem description**
Seen when testing https://github.com/phetsims/phet-io/issues/1626 in master:

In the Studio wrapper, the launched sim will produce sound when the Reset All button is pressed. It will not produce the Reset All button sound.

Also similar to what I reported for GFL https://github.com/phetsims/gravity-force-lab/issues/284#issuecomment-1048023810 and Faraday's Law https://github.com/phetsims/faradays-law/issues/222#issuecomment-1048025171.

I am not assigning to anyone at this time (MK's suggestion). 

**Steps to reproduce**
1. In Studio: generate sound by moving the slider for Voltage or Resistance.
2. Once the sound has stopped, launch the sim.
3. Press the reset button.

**Visuals**

https://user-images.githubusercontent.com/87318828/155189584-3932daee-d80a-47de-adfc-072cd8ef1f59.mov

**Comment by Nancy-Salpepi** [comment]
The behavior is not the same in the State wrapper: With Set State Rate = 0, if I generate sound in the downstream sim by moving the sliders and then press Reset All--no sound is generated. 

However, sound is generated if I:
1. Set state rate = 0
2. Generate sound in downstream sim by moving sliders
3. Press the "Set State Now" button

Also similar to what I see in GFL and Faraday's law.

**Comment by zepumph** [comment]
Yeah, this is one of those classic "hard state/sound" clashes to deal with. The model moved, but it isn't based on user input. Likely the solution is to have sound based on user-controlled elements instead of the model, so that when model Properties change because of state setting, we won't hear that sound. Thanks for the good labels, this should be tackled upon next work in the repo.

**Comment by arouinfar** [comment]
@jessegreenberg can you please take a look at this as part of the 1.5 release?
